lld/test/ELF/bp-section-orderer-errs.s

```diff
@@ -0,0 +1,45 @@
+# REQUIRES: aarch64
```

Comment:
I think you can move all these "is incompatible with" tests to https://github.com/llvm/llvm-project/blob/main/lld/test/ELF/incompatible.s
Just add `touch %t.call_graph` and `touch %t.profdata` to create the files. They won't be parsed so they can be empty. You also won't need `split-file`. That will remove a lot of the complexity of this test.